metadata.rb

```diff
@@ -3,7 +3,7 @@
 maintainer_email 'nitz.raz@gmail.com'
 license          'GPL-2.0'
 description      'Installs/Configures python and python accessories'
-version          '1.0.0'
+version          '1.0.99'
```

Comment:
```suggestion
version          '1.0.0'
```
Did you want to omit this bump here?

Reply:
Did this on purpose to show the version used in staging and it exists on chef server. While testing I find it easier to bump with the temp version.